sql/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -2333,6 +2344,7 @@ class SparkConnectPlanner(
     }
   }
 
+  @deprecated("TypedReduce is now implemented using a normal UDAF aggregator.", "4.0.0")
```

Comment:
Do we still have a test for this somewhere?

Reply:
No. Should we add one on the server side?

Reply:
We ought to for backwards compatibility. Let's try to do this in a follow-up.